app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx

```diff
@@ -42,6 +44,7 @@ function InPersonLocationPostOfficeSearchStep({ onChange, toPreviousStep, regist
       const { streetAddress, formattedCityStateZip } = selectedLocation;
       const selectedLocationAddress = `${streetAddress}, ${formattedCityStateZip}`;
       setSubmitEventMetadata({ selected_location: selectedLocationAddress });
```

Comment:
can we combine the 2 lines so it's : `setSubmitEventMetadata({selected_location: sla, in_person_cta_variant: ipcva})`